### ISSUE TITLE
Problem when renaming package using refactor

### ISSUE THREAD

**Issue by ngaud** [sarl/sarl#584]
A small error when I was renaming a package initially named "io.sarl.demos.basic.inheritance" into "io.sarl.demos.inheritance"

I obtain that
```
package io.sarl.demos.inheritancert io.sarl.core.AgentKilled
import io.sarl.core.AgentSpawned
```

instead of

```
package io.sarl.demos.inheritance
import io.sarl.core.AgentKilled
import io.sarl.core.AgentSpawned
```

Same thing when i try to rename "io.sarl.demos.inheritance" into io.sarl.article.inheritance"

I obtain this
```
package io.sarl.articles.inheritancence
import io.sarl.core.AgentKilled
```

**Comment by gallandarakhneorg** [comment]
Currently, only the package renaming from the package explorer is allowed. I will add the missed unit tests for this feature.

Renaming the package from the SARL editor will be fixed by Issue #585.

**Comment by gallandarakhneorg** [comment]
I cannot reproduce the issue with unit tests, the product launched from Eclipse IDE, or the product launched from the command line.
